pkg/plugin/processor/builtin/webhook/http.go

```diff
@@ -0,0 +1,306 @@
+// Copyright © 2024 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:generate paramgen -output=http_paramgen.go httpConfig
+
+package webhook
+
+import (
+	"bytes"
+	"context"
+	"io"
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/conduitio/conduit-commons/config"
+	"github.com/conduitio/conduit-commons/opencdc"
+	sdk "github.com/conduitio/conduit-processor-sdk"
+	"github.com/conduitio/conduit/pkg/foundation/cerrors"
+	"github.com/conduitio/conduit/pkg/foundation/log"
+	"github.com/goccy/go-json"
+	"github.com/jpillora/backoff"
+)
+
+type httpConfig struct {
+	// URL used in the HTTP request.
+	URL string `json:"request.url" validate:"required"`
+	// HTTP request method to be used.
+	Method string `json:"request.method" default:"POST"`
+	// Value of the Content-Type header.
```

Comment:
I believe go docs require us to start the comment with the variable name itself.
```suggestion
	// ContentType Value of the Content-Type header.
```
plz make sure all the others are updated too

Reply:
Done!